requests/exceptions.py

```diff
@@ -98,6 +98,9 @@ class StreamConsumedError(RequestException, TypeError):
 class RetryError(RequestException):
     """Custom retries logic failed"""
 
+class InvalidHeaderError(RequestException):
```

Comment:
This should be `ConflictingHeaderError`. The headers themselves are not invalid, they're an invalid combination, which you even mention in the docstring. Let's not tempt people to start using this to validate headers or any other such nonsense.